### DISCUSSION TITLE
LUA>> [DC4] What does this mean?

### DISCUSSION THREAD

**Discussion by bijlevel** [RomanKubiak/ctrlr#222] [Q&A]
Console displays: LUA>> [DC4] and most of the time several lines of the same output. What does this mean?

**Comment by Ted-juh** [comment]
Going over the problem mtarenskeen has with allowed characters in a listbox, I saw a mention of DC4 in the ctrlrUtilities.cpp.

DC4 is short for Device Control 4 and is one of the control characters in ascii, ascii 20 to be exact. In the ctrlrUtilities.cpp it's defined as an asciiSpecial. How or why you get that output, I don't know.
